.travis.yml

```diff
@@ -129,9 +129,12 @@ matrix:
                CONDA_VERSION=4.3.21
 
         # Try pre-release version of Numpy without optional dependencies
+        # Run this test using native pytest
         - os: linux
           env: NUMPY_VERSION=prerelease
```

Comment:
don't pick this one, as this hardly every runs...

Reply:
oops...

Reply:
which one do you think I should change?

Reply:
Well, feel free to modify any as it should be totally transparent and not matter, just not this one (as it only runs when there is a numpy prerelease, a few weeks in a year).